wrapper/smriprep_docker.py

```diff
@@ -222,6 +222,22 @@ def _get_posargs(usage):
 def get_parser():
     """Defines the command line interface of the wrapper"""
     import argparse
+    from functools import partial
+
+    class ToDict(argparse.Action):
+        def __call__(self, parser, namespace, values, option_string=None):
+            d = {}
+            for kv in values:
+                k, v = kv.split("=")
+                d[k] = os.path.abspath(v)
+            setattr(namespace, self.dest, d)
+
+    def _is_file(path, parser):
+        """Ensure a given path exists and it is a file."""
+        path = os.path.abspath(path)
+        if not os.path.isfile(path):
+            raise parser.error("Path should point to a file (or symlink of file): <%s>." % path)
+        return path
```

Comment:
not currently used?

Reply:
Updated to use it. I assume it solved a problem in fmriprep-docker, so might as well have it here...